### ISSUE TITLE
Better support diff-mode and magit.

### ISSUE THREAD

**Issue by tee3** [the-frey/cyberpunk-2019#7]
It seems like cyberpunk-2019 just uses the default diff-mode colors and the magit-diff (at least) is also not colored according to the theme.

**Comment by the-frey** [comment]
Hey, thanks for the feedback!

Yeah, I haven't added support for those as I don't personally use them. I'd welcome a PR!

**Comment by tee3** [comment]
I'll give it a shot.  Do you have a recommendation for colors for "added" / "deleted" / "changed"?  If not, I'll just make a proposal and you can judge it.

**Comment by tee3** [comment]
I'll give it a shot.  Do you have a recommendation for colors for "added" / "deleted" / "changed"?  If not, I'll just make a proposal and you can judge it.

**Comment by the-frey** [comment]
I would use the emacs rainbow mode to have a look at the colours in the scheme in the theme file.

Broadly, I guess I'd want to see:

- additions be one of the green options, but not the main text colour or one of the ones too close to it as that would be confusing
- changes be one of the rich blues so they're nice and clear. Alternatively there's some oranges you could use? up to you
- you could probably get away with the main bright pink/red for deleted, it's a good eye-catching shade

That's off the top of my head, based roughly on what the joker (linting) styling is for the theme. Give it a try!

**Comment by the-frey** [comment]
Can't 100% remember if these are the ones I use on mac as well, but outside the theme I have joker configured in my init file with something like

```
(set-face-attribute 'flycheck-error nil :underline '(:color "#FF4081"))
(set-face-attribute 'flycheck-warning nil :underline '(:color "#FF9C00"))
(set-face-attribute 'flycheck-info nil :underline '(:color "#9C00FF"))
```

**Comment by tee3** [comment]
Closed by #10.
